src/platform/BUILD.gn

```diff
@@ -44,6 +44,9 @@ if (chip_device_platform != "none") {
 
     # By pass provision and secure session
     chip_bypass_rendezvous = false
+
+    # Bypass including the additional data in the advertisement packets
```

Comment:
some information of what 'addiotional data' means would be nice.
Maybe this should instead be focused on 'rotating device id' instead of additional data. Additional data seems vague.

Reply:
i added some description to AdditionalDataPayload.h